src/libraries/listener.library.ts

```diff
@@ -1,11 +1,29 @@
 import { BlackListedKeyboardShortcutChecker } from './black-listed-key-bindings.library';
 
-export interface IKeyboardShortcutListenerOptions {
+export interface IKeyboardShortcutListenerOptions { // defaults with ?
+    /**
+     * @property {KeyboardShortcutCombination}
+     */
     keyBinding: KeyboardShortcutCombination;
+    /**
+     * @property {string} to explain to the user what the shortcut combination you made will do
+     */
     description: string;
+    /**
+     * @property { number } the order of presedence when more than one key matches
+     */
     priority?: number;
+    /**
+     * @property { boolean } if this is true, and you are inside a text box and do shortcut it wouldn't actually fire
```

Comment:
this description is wrong for this property.  Should be something like "if this is true, and more than one keybinding matches, then after resolving the first keybinding it will be passed on to the next keybinding so it can also be resolved"